group/exclude/schema.js

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: 2022 Mix Irving
+//
+// SPDX-License-Identifier: LGPL-3.0-only
+
+const { tangle, feedId, groupId } = require('../definitions')
+
+module.exports = {
+  type: 'object',
+  required: ['type', 'excludes', 'recps', 'tangles'],
+  properties: {
+    type: {
+      type: 'string',
+      pattern: '^group/exclude$'
+    },
+    excludes: {
+      type: 'array',
+      items: [{ $ref: '#/definitions/feedId' }],
+      minItems: 1,
+      maxItems: 16
+    },
```

Comment:
I think we want to support adding a `comment: String` to this message? (optional)

Reply:
future work?